frame/composable-tests-helpers/src/test/proptest.rs

```diff
@@ -36,3 +36,36 @@ macro_rules! prop_assert_acceptable_computation_error {
 		);
 	}};
 }
+
+/// Accepts -2, -1, 0, 1, 2
+#[macro_export]
+macro_rules! prop_assert_zero_epsilon {
+	($x:expr) => {{
+		let epsilon = 2;
+		let upper = 0 + epsilon;
+		let lower = 0;
+		prop_assert!(upper >= $x && $x >= lower, "{} => {} >= {}", upper, $x, lower);
+	}};
+}
+
+/// Accept a 'dust' deviation
+#[macro_export]
+macro_rules! prop_assert_epsilon {
+	($x:expr, $y:expr) => {{
+		let precision = 1000;
+		let epsilon = 5;
+		let upper = precision + epsilon;
+		let lower = precision - epsilon;
+		let q = multiply_by_rational($x, precision, $y).expect("qed;");
+		prop_assert!(
+			upper >= q && q >= lower,
+			"({}) => {} >= {} * {} / {} >= {}",
+			q,
+			upper,
+			$x,
+			precision,
+			$y,
+			lower
+		);
+	}};
+}
```

Comment:
We don't need this function anymore?

Reply:
I require it ,as other functions does not allow `q` in range 0.995 to 1.005. For exchange with high slippage i.e almost emptying the pool I need this.

Reply:
I was meaning that there is already a function in the crate that is doing that

Reply:
both are slightly different than each other.